packages/postgres/src/default-pool.ts

```diff
@@ -1,7 +1,7 @@
-import { PostgresPool } from './pool';
+import { PostgresPool, type CursorIterator, type QueryParams } from './pool';
```

Comment:
You'll want to run `make changeset` and add a changeset so that a new version of the package is published for consumption in PT.

Reply:
Done in 4cee2e37f.